synapse/storage/databases/main/room.py

```diff
@@ -1569,6 +1622,69 @@ def _background_populate_rooms_creator_column_txn(
 
         return batch_size
 
+    async def _background_add_room_type_column(
+        self, progress: JsonDict, batch_size: int
+    ) -> int:
```

Comment:
Not really sure where to make this thread, so I'll do it here (I didn't see this previously discussed on the PR, but do let me know if I missed it!)

Since this is a background update I'm assuming the new `room_type` column is `null` for all rows until we fill in it. This will cause incorrect results to be returned until the background update is complete -- do we think this is "fine" until it is done?

Reply:
I think it is "fine" though I am not sure if I should be the one to judge that

Not sure if this could be avoided anyway

Reply:
I think for this use-case it is most likely fine too, just was curious if I had missed a conversation about it.